author/backends.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
But this is UNIX notation of executable script. Since the `backends.py` file is not intended to run standalone, I would prefer either to delete this line completely or to leave it unchanged.